README.md

```diff
@@ -1 +1,41 @@
-# Hello_Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+>Hello Microverse is my firt project done on microverse
+## Built With
+
+- HTML
+- CSS
+- Mark down
+
+## Getting Started
+
+**clone this git repository.**
+
```

Comment:
- [ ] I would recommend you make your README documentation as detailed as possible. Kindly include detailed instructions on how users can set up and run your project locally. For example:
  - Clone this repo by running `git clone https://github.com/Brudaman/Hello_Microverse.git`
  - Change into directory: `cd Hello_Microverse`
  - Open `index.html` in the browser

- [ ] Going forward I would recommend you make your commit messages more descriptive. 'Thank you' is rather vague and does not describe the feature you wish to commit. From the [Microverse lesson here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) you can see that your commit should be in the  imperative and capitalized.

  Example of a good commit message
  - 'Change title color to red'
  - You can see more from the link shared above.

- [ ] For subsequent projects I would recommend you name your branches something more descriptive. For example, instead of `project`, you may name your branch `hello_microverse` since because it describes your project more.